packages/core/src/react/blocks/YoutubeLiteBlock.tsx

```diff
@@ -55,11 +56,11 @@ export function YoutubeLiteBlock({ domNode }: Omit<IYoutubeLiteBlock, 'component
 		}
 	}
 
-	const { src, title } = attribs;
+	const { src, title, params = '' } = attribs;
```

Comment:
Looks like youtube lite uses `autoplay=1` as a default. So either we don't set params if nothing is set (and omit it entirely) or we make our `params` default to `autoplay=1`. This would ensure backwards compatibility

Can you also update the tests, some snapshots are failing bc we're not always passing params.